mathjax3-ts/core/OutputJax.ts

```diff
@@ -154,4 +162,19 @@ export abstract class AbstractOutputJax<N, T, D> implements OutputJax<N, T, D> {
         return null as N;
     }
 
+    /*
+     * Execute a set of filters, passing them the MathItem and any needed data,
+     *  and return the (possibly modified) data
+     *
+     * @param{FunctionList} filters  The list of functions to be performed
+     * @param{MathItem} math         The math item that is being processed
+     * @param{any} data              Whatever other data is needed
```

Comment:
Should those be rest arguments ...data: any[]?

Reply:
No, the data is used as part of an object in the function below, and if it were `...data: any[]`, then that would always be an array, and the functions that are being executed would have to be changed to deal with the array (instead of just the data item as they currently are).  If you want an array of items, then you can pass an array as `data`.  I don't want to force it to be an array (in fact, at the moment, only one item is passed, so forcing an array would make things harder to use).